llvm/lib/Target/AArch64/CMakeLists.txt

```diff
@@ -107,6 +107,7 @@ add_llvm_target(AArch64CodeGen
   Core
   GlobalISel
   MC
+  Passes
```

Comment:
Why is this adding a new dependency?

Reply:
`OptimizationLevel::O0` is a variable defined by Passes .cpp file. This is needed to avoid an undefined symbol linker error. I don't find a way to define a `static constexpr variable ` of type OptimizationLevel within OptimizationLevel. Before `}`, the class `OptimizationLevel` is not considered a literal class.

```
struct A {
    int a, b;
    constexpr A(int a, int b) : a(a), b(b) {}
    
    static constexpr A default_value{0, 0};
};

// would cause
error: constexpr variable cannot have non-literal type 'const A'
    6 |   static constexpr inline A default_value{0, 0};
```

Reply:
BTW, it looks like #157057 may not go anywhere, so don't let me hold up this PR.